node/adapter/requirements.txt

```diff
@@ -15,6 +15,6 @@ pyOpenSSL==0.15.1
 pyasn1==0.1.7
 pyasn1-modules==0.0.5
 six==1.9.0
-urllib3==1.10.3
+urllib3>=1.7,<=1.8
```

Comment:
Should be `>=1.7,<1.8`  i.e. we want the newest 1.7.x release.  1.8 added new exceptions that python-etcd doesn't seem to be expecting.  1.9 removed those exceptions and added different ones.